var/ramble/repos/builtin/package_managers/pip/package_manager.py

```diff
@@ -152,7 +152,7 @@ def _software_install(self, workspace, app_inst=None):
                         )
                         and pkg not in installed_pkgs
                     ):
-                        logger.die(
+                        logger.warn(
```

Comment:
I need to figure out how a mechanism to designate which packages should be spack installed vs pip installed in the `ramble.yaml` to avoid this. 

Right now, this is the pip installer complaining about the `caliper` package not being installed, which makes sense why it is an error currently.

Reply:
This might help. https://ramble.readthedocs.io/en/latest/configuration_files.html#package-manager-specific-packages

If not, let me know.